### PR TITLE
Added Connection Settings Page

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

**Comment by thatarchguy** [comment]
The python looks good. The tests are beautiful

**Comment by mprahl** [comment]
@thatarchguy, cool! Let me finish writing the tests and then I'll merge it.

**Comment by mprahl** [comment]
@thatarchguy, I finished writing tests for what's written in the app so far. Please let me know what you think in the latest commit, and then I'll merge it.

File: adreset2/apiv1/ad_config.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+"""
+from flask import request
+from flask_login import login_required, current_user
+from adreset2 import db
+from adreset2.models import AdConfigs
+from adreset2.errors import ValidationError, GenericError
+from adreset2.decorators import json_wrap, paginate
+from adreset2.apiv1 import apiv1
+# This broad import statement is so that functions from utils can be mocked in the Blueprint
+import adreset2.utils
```
Comment (thatarchguy) [inline adreset2/apiv1/ad_config.py:14]: Can you explain this more to me? 

Is this because if you import the functions directly you can't mock them?

Comment (mprahl) [inline adreset2/apiv1/ad_config.py:14]: So you can when it is imported inside a Python module, but a Flask Blueprint is not a module, and ends up being an object. 
I posted a stackoverflow but haven't heard back:
http://stackoverflow.com/questions/37766533/how-to-mock-imported-function-in-a-flask-blueprint

Comment (mprahl) [inline adreset2/apiv1/ad_config.py:14]: @thatarchguy, thoughts?

**Pull request merged**